### PR TITLE
[Merged by Bors] - chore(Imo/Imo1998Q2): remove resolved porting notes

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Archive/Imo/Imo1998Q2.lean [hunk 1]
```diff
@@ -130,19 +130,17 @@ theorem A_fibre_over_contestant_card (c : C) :
       ((A r).filter fun a : AgreedTriple C J => a.contestant = c).card := by
   rw [A_fibre_over_contestant r]
   apply Finset.card_image_of_injOn
-  -- Porting note (#10936): used to be `tidy`. TODO: remove `ext` after `extCore` to `aesop`.
-  unfold Set.InjOn; intros; ext; all_goals aesop
+  -- Porting note (#10936): used to be `tidy`.
```
Comment (grunweg) [inline Archive/Imo/Imo1998Q2.lean:133]: I guess this porting note is no longer actionable (the `tidy` tactic was not ported; aesop is meant to replace it), so tend towards removing it. I'll let a maintainer comment.

**Review by grunweg — COMMENTED** [review]
Thank you for your contribution! This looks good to me; I wonder if the other porting note should *also* be removed - and would welcome the opinion of a maintainer here.
maintainer merge

File: Archive/Imo/Imo1998Q2.lean (hunk 1 above)
Comment (jcommelin) [reply Archive/Imo/Imo1998Q2.lean:133]: Yes, please remove it.

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Archive/Imo/Imo1998Q2.lean (hunk 1 above)
Comment (mo271) [inline Archive/Imo/Imo1998Q2.lean:133]:
```suggestion
```

Comment (mo271) [reply Archive/Imo/Imo1998Q2.lean]: done

**Pull request closed without merging**